spec/actors/hyrax/actors/generic_work_actor_spec.rb

```diff
@@ -204,11 +201,8 @@
   describe '#update' do
     let(:curation_concern) { create(:generic_work, user: user, admin_set_id: admin_set.id) }
 
-    subject { Hyrax::CurationConcern.actor }
-
     context 'failure' do
       let(:attributes) { {} }
-
```

Comment:
@jcoyne We can deal with that when we get to later versions of rubocop-rspec with one big autocorrect, if we care.  For this terse little `let`/`it` pair though, I like it together.